src/components/card/_card.scss

```
@@ -149,6 +153,10 @@
     border-top-left-radius: $euiBorderRadius - 1px;
     border-top-right-radius: $euiBorderRadius - 1px;
 
+    & img {
+      width: 100%;
+    }
```

Not necessary to display the `&` for simple nesting. This can be confusing too when scanning and thinking it's trying to concatenate onto something.
```suggestion
    img {
      width: 100%;
    }
```

---

My mistake